packages/nimble-components/src/rich-text/editor/template.ts

```diff
@@ -84,12 +90,37 @@ export const template = html<RichTextEditor>`
                         ${x => richTextToggleNumberedListLabel.getValueFor(x)}
                         <${iconNumberListTag} slot="start"></${iconNumberListTag}>
                     </${toggleButtonTag}>
+                    ${repeat(
+        x => x.getMentionExtensionConfig(),
+        html<MentionExtensionConfiguration, RichTextEditor>`<${buttonTag}
+                        appearance="ghost"
+                        content-hidden
+                        ?disabled="${(_x, c) => c.parent.disabled}"
+                        slot="start"
+                        title=${x => x.label}
+                        @click=${(x, c) => c.parent.mentionButtonClick(x.character)}
+                    >
+                        ${x => x.label}
+                        ${x => x.iconTemplate}
+                    </${buttonTag}>`
+    )}
                 </${toolbarTag}>
                 <span class="footer-actions" part="footer-actions">
                     <slot name="footer-actions"></slot>
                 </span>
             </section>
             ${errorTextTemplate}
         </div>
+        <${richTextMentionListBoxTag}
+            ${ref('mentionListBox')}
+            @activate-mention=${(x, c) => x.activateMention(c.event as CustomEvent<MentionDetail>)}
```

Comment:
make sure to update the event name you bind to, i.e. mention-selected

Reply:
Updated